packages/eth-json-rpc-provider/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.1.0]
+
+### Added
+
+- Make `SafeEventEmitterProvider` EIP-1193 compatible by adding a `request` method ([#4422](https://github.com/MetaMask/core/pull/4422))
+  - Now `SafeEventEmitterProvider` is compatible with `@metamask/eth-query`, `@metamask/ethjs-query`, `BrowserProvider` from Ethers v6 and `Web3Provider` from Ethers v5
+
+### Changed
```

Comment:
I forgot — the [Keep a Changelog](https://keepachangelog.com/en/1.1.0/#how) spec designates a Deprecated section for us to use in these cases:

``` suggestion
### Deprecated
```